contracts/test-integration/BlsProviderContractInteraction.test.ts

```diff
@@ -7,10 +7,12 @@ import getNetworkConfig from "../shared/helpers/getNetworkConfig";
 
 describe("Provider tests", function () {
   let blsProvider;
+  let blsSigner;
   let fundedWallet: Wallet;
 
-  this.beforeAll(async function () {
+  this.beforeAll(async () => {
     const networkConfig = await getNetworkConfig("local");
+    const privateKey = ethers.Wallet.createRandom().privateKey;
```

Comment:
nit but we could start using the new random PK function for this.

```
const pk = await BlsWalletWrapper.getRandomBlsPrivateKey();
```

Reply:
Good shout, updated